src/metabase/db/liquibase.clj

```diff
@@ -254,7 +255,7 @@
      :or {change-set-filters []}}]
    (let [change-log     (.getDatabaseChangeLog liquibase)
          database       (.getDatabase liquibase)
-         log-iterator   (ChangeLogIterator. change-log (into-array liquibase.changelog.filter.ChangeSetFilter change-set-filters))
+         log-iterator   (ChangeLogIterator. change-log (into-array ChangeSetFilter change-set-filters))
```

Comment:
Made this change while trying to debug the Liquibase issues (unsuccessfully, so far) but didn't see a reason to revert it